instrumentation/slowrendering/src/main/java/io/opentelemetry/android/instrumentation/slowrendering/SlowRenderingInstrumentation.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.android.instrumentation.slowrendering;
+
+import android.app.Application;
+import android.os.Build;
+import android.util.Log;
+import androidx.annotation.NonNull;
+import androidx.annotation.RequiresApi;
+import io.opentelemetry.android.OpenTelemetryRum;
+import io.opentelemetry.android.common.RumConstants;
+import io.opentelemetry.android.instrumentation.AndroidInstrumentation;
+import java.time.Duration;
+
+/** Entrypoint for installing the slow rendering detection instrumentation. */
+public final class SlowRenderingInstrumentation implements AndroidInstrumentation {
+
+    Duration slowRenderingDetectionPollInterval = Duration.ofSeconds(1);
+
+    /**
+     * Configures the rate at which frame render durations are polled.
+     *
+     * @param interval The period that should be used for polling.
+     * @return {@code this}
+     */
+    public SlowRenderingInstrumentation setSlowRenderingDetectionPollInterval(Duration interval) {
```

Comment:
Presumably the agent detects when this class is on the classpath, instantiates it, and installs it. How would we expect this method to ever get called? I think there are no usages outside of tests.

Reply:
@breedx-splk I'm a bit worried about not having all the parts ready before the next release, since the agent won't have any auto instrumentation working until all parts are done, so I'll go ahead and merge this PR and we can continue this discussion on the SIG if you'd like as this way of configuring instrumentations is not exclusive to the slowrendering one anyway.